test/libsolidity/syntaxTests/emit/emit_struct_memory.sol

```diff
@@ -0,0 +1,9 @@
+contract c {
+    struct S { uint a ; }
+    event E(S);
+    function f() public {
+        emit E(S(6));
```

Comment:
Perhaps we should remove this line (and also the function) to have a minimal test case?

Reply:
(also applies to the other test cases)